<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1223</width>
    <height>689</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Protein Digester</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/resources.qrc">
    <normaloff>:/pixmap/icons/96x96/applications-science.png</normaloff>:/pixmap/icons/96x96/applications-science.png</iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QSplitter" name="mainSplitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="opaqueResize">
       <bool>true</bool>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="mainSplitterTopWidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="proteinsTitleLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Proteins&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="proteinsSearchWidget" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <property name="spacing">
            <number>6</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLineEdit" name="proteinsSearchLineEdit">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="placeholderText">
              <string/>
             </property>
             <property name="clearButtonEnabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="proteinsSearchTypeComboBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="sizeAdjustPolicy">
              <enum>QComboBox::AdjustToContents</enum>
             </property>
             <item>
              <property name="text">
               <string>By name</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>By sequence</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>By digest peptide</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="proteinsSearchPushButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="icon">
              <iconset resource="../resources/resources.qrc">
               <normaloff>:/pixmap/icons/16x16/edit-find-symbolic.symbolic.png</normaloff>:/pixmap/icons/16x16/edit-find-symbolic.symbolic.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>16</width>
               <height>16</height>
              </size>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="proteinsTableWidget">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="verticalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
          <property name="horizontalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
          <property name="showGrid">
           <bool>false</bool>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <property name="cornerButtonEnabled">
           <bool>false</bool>
          </property>
          <property name="columnCount">
           <number>2</number>
          </property>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>#</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Name</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="mainSplitterBottomWidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>6</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QWidget" name="subSplitterLeftWidget" native="true">
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="spacing">
             <number>6</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="peptideTitleLabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Peptides&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTableWidget" name="peptidesTableWidget">
              <property name="editTriggers">
               <set>QAbstractItemView::NoEditTriggers</set>
              </property>
              <property name="alternatingRowColors">
               <bool>true</bool>
              </property>
              <property name="selectionMode">
               <enum>QAbstractItemView::SingleSelection</enum>
              </property>
              <property name="selectionBehavior">
               <enum>QAbstractItemView::SelectRows</enum>
              </property>
              <property name="verticalScrollMode">
               <enum>QAbstractItemView::ScrollPerPixel</enum>
              </property>
              <property name="horizontalScrollMode">
               <enum>QAbstractItemView::ScrollPerPixel</enum>
              </property>
              <property name="showGrid">
               <bool>false</bool>
              </property>
              <property name="sortingEnabled">
               <bool>true</bool>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
              <property name="cornerButtonEnabled">
               <bool>false</bool>
              </property>
              <attribute name="horizontalHeaderStretchLastSection">
               <bool>true</bool>
              </attribute>
              <attribute name="verticalHeaderVisible">
               <bool>false</bool>
              </attribute>
              <column>
               <property name="text">
                <string>#</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Sequence</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Missed cleavages</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Unique</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="subSplitterRightWidget" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="subProteinsTitleLabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Proteins&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTableWidget" name="subProteinsTableWidget">
              <property name="editTriggers">
               <set>QAbstractItemView::NoEditTriggers</set>
              </property>
              <property name="alternatingRowColors">
               <bool>true</bool>
              </property>
              <property name="selectionMode">
               <enum>QAbstractItemView::SingleSelection</enum>
              </property>
              <property name="selectionBehavior">
               <enum>QAbstractItemView::SelectRows</enum>
              </property>
              <property name="verticalScrollMode">
               <enum>QAbstractItemView::ScrollPerPixel</enum>
              </property>
              <property name="horizontalScrollMode">
               <enum>QAbstractItemView::ScrollPerPixel</enum>
              </property>
              <property name="showGrid">
               <bool>false</bool>
              </property>
              <property name="sortingEnabled">
               <bool>true</bool>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
              <property name="cornerButtonEnabled">
               <bool>false</bool>
              </property>
              <attribute name="horizontalHeaderStretchLastSection">
               <bool>true</bool>
              </attribute>
              <attribute name="verticalHeaderVisible">
               <bool>false</bool>
              </attribute>
              <column>
               <property name="text">
                <string>#</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Name</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1223</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="locale">
     <locale language="French" country="France"/>
    </property>
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="createDatabaseAction"/>
    <addaction name="openDatabaseAction"/>
    <addaction name="separator"/>
    <addaction name="quitAction"/>
   </widget>
   <widget class="QMenu" name="databaseMenu">
    <property name="title">
     <string>Digestion database</string>
    </property>
    <addaction name="importFastaAction"/>
    <addaction name="addDigestionAction"/>
   </widget>
   <widget class="QMenu" name="workingDigestionMenu">
    <property name="title">
     <string>Working digestion</string>
    </property>
   </widget>
   <addaction name="fileMenu"/>
   <addaction name="databaseMenu"/>
   <addaction name="workingDigestionMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="createDatabaseAction">
   <property name="text">
    <string>Create digestion database</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="openDatabaseAction">
   <property name="text">
    <string>Open digestion database</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="importFastaAction">
   <property name="text">
    <string>Import FASTA file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="quitAction">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="addDigestionAction">
   <property name="text">
    <string>Add digestion</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>quitAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>548</x>
     <y>373</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>createDatabaseAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>createDatabaseActionTriggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>548</x>
     <y>373</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>openDatabaseAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>openDatabaseActionTriggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>548</x>
     <y>373</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>importFastaAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>importFastaActionTriggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>548</x>
     <y>373</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>proteinsSearchPushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>proteinsSearchPushButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1071</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>548</x>
     <y>373</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>proteinsSearchLineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>proteinsSearchPushButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>423</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>1071</x>
     <y>78</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addDigestionAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>addDigestionActionTriggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>548</x>
     <y>373</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>proteinsTableWidget</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>MainWindow</receiver>
   <slot>proteinsTableWidgetItemSelectionChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>611</x>
     <y>268</y>
    </hint>
    <hint type="destinationlabel">
     <x>611</x>
     <y>344</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>peptidesTableWidget</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>MainWindow</receiver>
   <slot>peptidesTableWidgetItemSelectionChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>308</x>
     <y>562</y>
    </hint>
    <hint type="destinationlabel">
     <x>611</x>
     <y>344</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>createDatabaseActionTriggered()</slot>
  <slot>importFastaActionTriggered()</slot>
  <slot>openDatabaseActionTriggered()</slot>
  <slot>proteinsSearchPushButtonClicked()</slot>
  <slot>addDigestionActionTriggered()</slot>
  <slot>proteinsTableWidgetItemSelectionChanged()</slot>
  <slot>peptidesTableWidgetItemSelectionChanged()</slot>
 </slots>
</ui>
